src/main/java/eu/dissco/core/datacitepublisher/domain/datacite/UriScheme.java

```diff
@@ -0,0 +1,38 @@
+package eu.dissco.core.datacitepublisher.domain.datacite;
+
+public enum UriScheme {
+  ROR("https://ror.org", "ROR"),
+  HANDLE("https://hdl.handle.net", "Handle"),
+  DOI("https://doi.org", "DOI"),
+  QID(null, "Q Number");
```

Comment:
Can't this be `https://www.wikidata.org/` ?

Reply:
right I was under the impression this was a vocabulary, but it's free text. changed